### ISSUE TITLE
Update to crystal v0.27.0

### ISSUE THREAD

**Issue by Jens0512** [icyleaf/popcorn#4]
Build fails in (newly released) crystal version 0.27.0:
```
% crystal version
Crystal 0.27.0 (2018-11-02)

LLVM: 6.0.1
Default target: x86_64-pc-linux-gnu
% crystal spec # In icyleaf/popcorn (master branch)
 
 ... ... ... # Long boring stack-trace

in src/popcorn/cast/int.cr:104: undefined constant UNIX_SECONDS (
did you mean 'MAX_SECONDS')

    seconds = UNIX_SECONDS + (milliseconds / 1_000)
```

**Comment by icyleaf** [comment]
Yeah, i'll follow it.

**Comment by icyleaf** [comment]
Released `v0.2.1`

**Comment by Jens0512** [comment]
Thanks!
